Sources/System/Internals/Syscalls.swift

```diff
@@ -191,7 +191,7 @@ internal func system_confstr(
 internal let SYSTEM_AT_REMOVE_DIR = AT_REMOVEDIR
 internal let SYSTEM_DT_DIR = DT_DIR
 internal typealias system_dirent = dirent
-#if os(Linux) || os(Android) || os(FreeBSD)
+#if os(Linux) || os(Android) || os(FreeBSD) || os(OpenBSD)
 internal typealias system_DIRPtr = OpaquePointer
 #else
 internal typealias system_DIRPtr = UnsafeMutablePointer<DIR>
```

Comment:
What exactly is "else" here? I'm guessing only Darwin falls here in practice. Would this be more stable to accommodate new platforms if written as if-Darwin-else?

Reply:
Hmm, seems like it, given the if !Windows above. Anyways, understandable to leave it for now.